packages/contracts-bedrock/test/L2Genesis.t.sol

```diff
@@ -135,8 +135,8 @@ contract L2GenesisTest is Test {
         // 2 predeploys do not have proxies
         assertEq(getCodeCount(_path, "Proxy.sol:Proxy"), Predeploys.PREDEPLOY_COUNT - 2);
 
-        // 17 proxies have the implementation set
```

Comment:
We don't want to add this to genesis yet

Reply:
this would make L2Genesis `test_genesis_predeploys` fail. should I still go ahead?

Reply:
```
Failing tests:
Encountered 1 failing test in test/L2Genesis.t.sol:L2GenesisTest
[FAIL. Reason: assertion failed: 18 != 17] test_genesis_predeploys() (gas: 13077574)
```

Reply:
not sure I understand. can you provide more details?